src/test/java/io/confluent/connect/elasticsearch/ValidatorTest.java

```diff
@@ -111,6 +113,31 @@ public void testValidCredentials() {
     assertNoErrors(result);
   }
 
+  @Test
+  public void testValidBothDataStreamConfigs() {
+    props.put(DATA_STREAM_DATASET_CONFIG, "a_valid_dataset");
+    props.put(DATA_STREAM_TYPE_CONFIG, "logs");
+    validator = new Validator(props, () -> mockClient);
+    Config result = validator.validate();
+    assertNoErrors(result);
+  }
+
+  @Test
+  public void testValidMissingBothDataStreamConfigs() {
+    validator = new Validator(props, () -> mockClient);
+    Config result = validator.validate();
+    assertNoErrors(result);
+  }
```

Comment:
```suggestion
  @Test
  public void testValidDefaultConfig() {
    validator = new Validator(props, () -> mockClient);
    Config result = validator.validate();
    assertNoErrors(result);
  }
```
nice catch for a test that we are missing, but this is more testing that all of our default configs are valid i.e missing datastream ones